src/bin/rustfmt.rs

```diff
@@ -120,6 +120,8 @@ fn make_opts() -> Options {
     opts.optflag("h", "help", "show this message");
     opts.optflag("V", "version", "show version information");
     opts.optflag("v", "verbose", "print verbose output");
+    // Supress warning. Remove this option after the default write mode changed to overwrite.
+    opts.optflag("", "no-warn", "inhibit warning about write-mode change");
```

Comment:
could you make the name a bit more specific please? Perhaps "no-warn-write-mode", but given the verbosity, we could allow `-w` too.